### ISSUE TITLE
Remove obsolete flag from Azure.Preview_2021_12

### ISSUE THREAD

**Issue by BernieWhite** [Azure/PSRule.Rules.Azure#1166]
**Description of the issue**

The baseline `Azure.Preview_2021_12` is incorrectly flagged obsolete.

**Module in use and version:**

- Module: PSRule.Rules.Azure
- Version: **1.11.0-B2112073**

Captured output from `$PSVersionTable`:

```text
Name                           Value
----                           -----
PSVersion                      7.2.1
PSEdition                      Core
GitCommitId                    7.2.1
OS                             Microsoft Windows 10.0.22000
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```
